arm/Microsoft.AppConfiguration/configurationStores/keyValues/.bicep/nested_rbac.bicep

```diff
@@ -0,0 +1,55 @@
+@sys.description('Required. The IDs of the principals to assign the role to.')
```

Comment:
Please rename this file to `nested_roleAssignments`

Reply:
Changed the file name from nested_rbac.bicep to nested_roleAssignments.bicep